tests/unit/test_url_utils.py

```
@@ -0,0 +1,28 @@
+    """ホストとパスは保持される。"""
+    url = "http://sub.example.co.jp:8080/api/v1/data"
+    assert mask_url(url) == url
+
+
+def test_mask_url_empty_string() -> None:
+    """空文字列は空文字列を返す。"""
+    assert mask_url("") == ""
+
+
+def test_mask_url_invalid_input() -> None:
+    """無効な URL 形式の場合、'invalid-url' を返す（例外を投げない）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)